addOns/reports/src/main/zapHomeFiles/reports/traditional-pdf/report.html

```diff
@@ -179,7 +179,8 @@ <h3 th:text="#{report.alerts.list}">Alerts</h3>
 			</tr>
 			<tr th:each="alert: ${alertTree.children}">
 				<td><a th:href="'#' + ${alert.userObject.pluginId}"
-					th:text="${alert.nodeName}" href="#oluginId">Alert Name</a></td>
+					th:text="${helper.escapeXml(alert.nodeName)}" href="#oluginId">Alert
```

Comment:
Should the href really be olugin not plugin? I know that was there before but it seems out of place.